index.html

```diff
@@ -272,5 +272,23 @@ <h3 class="sub_title">
         </ul>
       </section>
     </main>
+    <!-- contact form section  -->
+
+    <section class="contact-form">
+     <div class="container">
+      <div class="contact-text">
+        <h2>Contact me</h2>
+        <p>If you have an application you are interested in developing, 
+          a feature that you need built or a project that needs coding.
+           I’d love to help with it </p>
+       </div>
+       <form class="forms">    
```

Comment:
I suggest you add the [formspree](https://formspree.io/html/) service to ensure your form submits. Its part of the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md)
<img width="1518" alt="Screenshot 2022-11-01 at 14 57 53" src="https://user-images.githubusercontent.com/48546992/199228107-fcb209e0-cbdc-4993-a831-0d049fb0e30c.png">